src/index.html

```diff
@@ -1,17 +1,229 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <h1>HTML Forms</h1>
+
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fields">
+        <legend>Personal information</legend>
+
+
+        <div class = "form__items">
+          <label for="Surname">
+          Surname:
+          </label>
+          <input 
+          type="text" 
+          name="Surname" 
+          id="Surname"
+          required 
+          autocomplete="off"
```

Comment:
fix indents in all places
```suggestion
          <input 
            type="text" 
            name="Surname" 
            id="Surname"
            required 
            autocomplete="off"
```